editor/sidebar/index.js

```diff
@@ -4,19 +4,32 @@
 import { Slot } from 'react-slot-fill';
 
 /**
+ * WordPress dependencies
+ */
+import Panel from 'components/panel';
+import PanelHeader from 'components/panel/header';
+import PanelBody from 'components/panel/body';
+
+/**
  * Internal Dependencies
  */
+import PostSettings from './post-settings';
 import './style.scss';
 
 const Sidebar = () => {
 	return (
 		<div className="editor-sidebar">
-			<div className="editor-sidebar__header">
-				<span className="editor-sidebar__select-post">Post</span> → Block
-			</div>
-			<div className="editor-sidebar__content">
-				<Slot name="Sidebar.Inspector" />
-			</div>
+			<PostSettings />
+			<Panel>
+				<PanelHeader>
+					<strong>
```

Comment:
Should we use the new `label` prop, or did you have challenges with the custom className? I'd expect `label={ <span ... /> }` should work well enough.